### DISCUSSION TITLE
FundMe.sol error- lesson 7

### DISCUSSION THREAD

**Discussion by Quadrhyno** [Cyfrin/foundry-full-course-cu#924] [Q&A]
I am getting this particular error when trying to compile the code on lesson 7
The error:  Wrong argument count for function call: 0 arguments given but expected 1.
  --> src/FundMe.sol:26:17:
   |
26 |         require(msg.value.getConversionRate() >=MINIMUM_USD, "didnt send enough ETH");
   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   
   The code i used is directly from GitHub according to the instructor.

**Comment by theirrationalone** [comment]
1. Make sure you have type directive `using PriceConverter for uint256;` at the top of your fundme contract. 

2. Make sure you're deploying latest compiled fundMe contract. (Most favourable in your case) 

3. Somehow... 

If your `getConversionRate(uint256,address)` function code in PriceConverter expecting priceFeedAddress as an arg then please provide that too i.e. `msg.value.getConversionRate(pricefeedAddress)`

;`)

**Comment by Loupsteb** [comment]
Hi,
i've exactly the same probleme, i dont uderstand why its working for Patrick in his vdeo but not for us with the code from the Lesson 4.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/5572b7a3-b129-4dee-b830-d8619f5e6ddd)

**Reply by Loupsteb** [reply]
finnaly ive comment this line :         require(msg.value.getConversionRate() >= MINIMUM_USD, "You need to spend more ETH!");
